application/modules/statements/views/view.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/*
+ * @var Client client
+ */
+
+// $cv = $this->controller->view_data["custom_values"];
+
+// print_r($quote);
+?>
+<script>
+
+// 	function submitForm(){
+// 		document.form1.target = "myActionWin";
+// 		window.open("","myActionWin");
+// 		document.form1.submit();
+// 	}
+
+    $(function () {
+
+        $('#btn_generate_pdf').click(function () {
+
+            $('#statement_form').submit();
+
+        	// submitForm();
+
+           // window.open('<?php echo site_url('statement/generate_pdf/' . '') ; // $quote_id); ?>', '_blank');
+        });
+
+        $(document).ready(function () {
+
+
+        });
+
+//         var fixHelper = function (e, tr) {
+//             var $originals = tr.children();
+//             var $helper = tr.clone();
+//             $helper.children().each(function (index) {
+//                 $(this).width($originals.eq(index).width());
+//             });
+//             return $helper;
+//         };
+//
+//         $('#item_table').sortable({
+//             helper: fixHelper,
+//             items: 'tbody',
+//         });
+    });
+</script>
+
+<div id="headerbar">
+    <h1 class="headerbar-title">
+        <?php echo $statement_number; ?>
+    </h1>
+
+    <div class="headerbar-item pull-right">
+        <div class="btn-group btn-group-sm">
+            <a class="btn btn-default dropdown-toggle" data-toggle="dropdown" href="#">
+                <?php _trans('options'); ?> <i class="fa fa-chevron-down"></i>
+            </a>
+            <ul class="dropdown-menu dropdown-menu-right">
+
+                <li>
+                    <a href="#" id="btn_generate_pdf"
```

Comment:
@pradeshc You may move the generation button into the properties box to the right of the client details